files/ko/web/media/formats/video_codecs/index.md

```diff
@@ -872,16 +872,27 @@ HEVC는 상용 포맷이며 여러 특허로 보호받고 있습니다. [MPEG LA
               <td>
                 11<sup><a href="#hevc-foot--1">[1]</a></sup>
               </td>
-              <td>No</td>
+              <td>94</td>
               <td>11</td>
             </tr>
           </tbody>
         </table>
+        <p>Chrome은 Windows 8+、Linux、하드웨어 코덱이 있는 ChromeOS 기기, 모든 macOS Big Sur+ 및 Android 5.0+ 기기에서 HEVC를 지원합니다.</p>
+        <p>Edge(Chromium)는 하드웨어 코덱이 있는 Windows 10 1709+ 장치에서 HEVC를 지원합니다(<a href="https://apps.microsoft.com/store/detail/hevc-video-extension/9NMZLZ57R3T7 "> 설치 필요) HEVC 비디오 확장 프로그램</a> 플러그인), 다른 운영 체제에 대한 지원은 Chrome과 일치합니다. Edge(레거시)는 HEVC 하드 디코딩만 지원합니다.
+        </p>
+        <p>HEVC는 특허로 보호되기 때문에 Mozilla는 HEVC를 지원하지 않습니다.</p>
+        <p>Internet Explorer는 하드웨어 코덱이 있는 장치에서 HEVC를 지원합니다.</p>
+        <p>Opera 및 기타 Chromium 기반 브라우저는 Chrome과 동일한 방식으로 지원됩니다.</p>
+        <p>Safari는 모든 macOS High Sierra+ 기기에서 HEVC를 지원합니다.</p>
```

Comment:
```suggestion
        <p>Safari는 macOS High Sierra 및 이후 버전의 모든 기기에서 HEVC를 지원합니다.</p>
```

> Safari supports HEVC for all devices on macOS High Sierra or later.

There doesn't seem to be a big difference in meaning, but I mentioned it because there is a little difference from the original text :)